### ISSUE TITLE
SE  Loss

### ISSUE THREAD

**Issue by purse1996** [zhanghang1989/PyTorch-Encoding#334]
论文中提到了SE Loss在解决一些小物体识别的准确性，但论文中没有给出loss的具体形式，也没有ablation study实验结果。想请教一下loss的具体形式

**Comment by zhanghang1989** [comment]
SE Loss 就是一个 multi-label classification 的 loss，binary cross-entropy 具体可以见代码。ablation study 在文章实验部分
